docs/reference/supported-environments.md

```diff
@@ -4,6 +4,12 @@ title: "Supported environments"
 description: "Find out where to run Camunda 8 components for SaaS and Self-Managed, including Optimize for both Camunda 8 and Camunda 7."
 ---
 
+The supported environments page lists tested and compatible environments, deployment options, and component dependencies for successfully running Camunda 8.
+
+**If the particular technology is not listed, we are unable to resolve issues or provide guidance on integration.** You may [raise a feature request](/contact) that will be evaluated by our product teams to provide official support from Camunda.
+
+Recommendations are denoted with [recommended], however, other options are supported as well.
```

Comment:
Maybe more verbose to ensure it is clearly understood what we mean:

```suggestion
If you started without any technology contraints, we advise you to follow our recommendations, which are denoted with [recommended]. However, the other listed options are supported as well.
```

Reply:
I like adding "the other listed options are supported as well," but I'm not sure we need the first half of the sentence to specify recommendations are for those without technology constraints. 

Since we use recommendations in other areas of our docs, we can add this as a glossary term and commit to what recommendations mean in the scope of Camunda docs.

Reply:
Okay. Can you please commit this proposal?